services/chaospod.go

```diff
@@ -188,14 +205,14 @@ func (m *chaosPodService) GenerateChaosPodsOfDisruption(instance *chaosv1beta1.D
 			pulseDormantDuration = instance.Spec.Pulse.DormantDuration.Duration()
 		}
 
-		notInjectedBefore := helpers.TimeToInject(instance.Spec.Triggers, instance.CreationTimestamp.Time)
+		notInjectedBefore := instance.TimeToInject(instance.CreationTimestamp.Time)
```

Comment:
Are we using something else than `instance.CreationTimestamp`?
If not maybe we could remove the arg also here

Reply:
Indeed, the `TimeToInject` is always used with the `timestamp` from the disruption itself. I refactor this method 👍